index.html

```diff
@@ -197,12 +197,12 @@ <h2>Photo Gallery</h2>
                             </select>
 
                             <label>Do you like radio buttons?</label>
-                            <label><input type='radio' name='radio' value='Yes'>Yes</label>
-                            <label><input type='radio' name='radio' value='No'>No</label>
+                            <label><input type='radio' name='radio' value='yes'>Yes</label>
+                            <label><input type='radio' name='radio' value='no'>No</label>
                             <label><input type='radio' name='radio' value='?' checked>None of your business</label>
 
-                            <label>What is the meaning of life?</label>
-                            <textarea name='life_meaning' rows='10'>Enter the correct answer here</textarea>
+                            <label for="life_meaning">What is the meaning of life?</label>
+                            <textarea name='life_meaning' id="life_meaning" rows='10' placeholder="Enter the correct answer here"></textarea>
```

Comment:
Missing 'for' attribute linking to id
Placeholders are a separate thing from values.  Placeholders are set using the placeholder attribute